### ISSUE TITLE
batchOperation/batch_operations_create_test && batch_operations_update_test -> should create a list of entities 001 && should update a list of entities 003

### ISSUE THREAD

**Issue by MohamedSadiq102** [Fiware/context.Orion-LD#303]
```
"body": {"detail": "urn:ngsi-ld:ContextSourceRegistration:csr1a3456", "title":
 "Registration already exists",
 
"type": "https://uri.etsi.org/ngsi-ld/errors/BadRequestData"}, 
"headers": {"connection": "close", "content-length": "158", 
"content-type": "application/json", 
"date": "Thu, 02 Jan 2020 16:14:26 GMT"}, "request": 

{"headers": {"Content-Type": "application/json", "accept": 
"application/json", "content-length": 526}, "method": "POST",
 "uri": {"auth": null, "hash": null, "host": 
"localhost:1026", "hostname": "localhost", 

"href": "http://localhost:1026/ngsi-ld/v1/csourceRegistrations/", "path": "/ngsi-
ld/v1/csourceRegistrations/", "pathname": "/ngsi-ld/v1/csourceRegistrations/",
 "port": "1026", "protocol": "http:", "query": null, 
"search": null, "slashes": true}}, "statusCode": 40

```

It expected 200 But it received 400

**Comment by kzangeli** [comment]
These two batch operations haven't been implemented.
Planned for Alpha 2 in June 2020

**Comment by MohamedSadiq102** [comment]
Fixed Issue #303
